hypervisor/start.c

```diff
@@ -250,6 +253,14 @@ static void init_palette(void)
 	/* static char <96>$<97>ß<82>Í<81>A<83>f<81>[<83>^<82>É<82>µ<82>©<8e>g<82>¦<82>È<82>¢<82>¯<82>ÇDB<96>$<97>ß<91><8a><93><96> */
 }
 
+static void init_pit(void)
+{
+    io_out8(PIT_CMD, 0x34);
+	io_out8(PIT_CH0, 0x9c);
+	io_out8(PIT_CH0, 0x2e);
+	return;
```

Comment:
Please fixup spacing. We agreed on 4 spaces instead of a tabs.

Reply:
OK